content/help/ops/setup/app-health-check/index.md

```diff
@@ -171,6 +171,24 @@ NAME                             READY     STATUS    RESTARTS   AGE
 liveness-http-975595bb6-5b2z7c   2/2       Running   0           1m
 {{< /text >}}
 
+#### Cleanup
+
+Remove the service and redeploy Istio with Helm option `sidecarInjectorWebhook.rewriteAppHTTPProbe=false`.
+ 
+1. To remove the service, run:
+{{< text bash >}}
+$ kubectl delete -f <(istioctl kube-inject -f @samples/health-check/liveness-command.yaml@)
+{{< /text >}}
+
+1. To reinstall Istio, run:
+{{< text bash >}}
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+    --set global.mtls.enabled=false --set sidecarInjectorWebhook.rewriteAppHTTPProbe=false \
+    -f install/kubernetes/helm/istio/values.yaml > $HOME/istio.yaml
+$ kubectl apply -f $HOME/istio.yaml
```

Comment:
Quick question, with this command line, did you notice the sidecar-injector gets restarted? I'm sure the configmap will be updated, but I'm not sure if sidecar injector will then watch out for that change.

In another words, with this istio-redeploy, the following `separate port` will work, and not get rewritten?

Reply:
> In another words, with this istio-redeploy, the following `separate port` will work, and not get rewritten?

That's how it worked for me. I'm pretty sure sidecar injector watches for changes, just needs a minute to update its config...

Reply:
To clarify: I tested reverting the config as described above, and then re-doing the "separate port" step.